### PR TITLE
Localization's Added

### PR DESCRIPTION
**Pull request by 65001**
Note : The localization were added by an automated script . 44 languages were added all in all. 
Including :

1.af_ZA (Afrikaans)
2.ar (Arabic)
3.bs-Latn (Bosnian (Latin) )
4.bg_BG (Bulgarian )
5.ca_ES (Catalan )
6.zh-CHS_CN (Chinese Simplified )
7.zh-CHT_CN (Chinese Traditoinal )
8.yue (Cantonese) 
9.hr_HR (Croatian )  
10.cs_CZ (Czech)
11.da_DK (Danish)
12.nl_Nl (Dutch)
13.et_EE (Estonian )
14.fi_FI (Finnish)
15.fr (French )
16.el_GR (Greek)
17.ht (Haitian Creole)
18.he_IL (Hebrew)
19.hi_In (Hindi)
20.mww (Hmong Daw)
21.hu_HU (Hungarian)
22.id_ID (Indonesian)
23.ja_JP (Japanese)
24.sw_KE (Kiswahili)
25.ko_KR (Korean )
26.lv_LV (Latvian)
27.lt_LT (Lithuanian)
28.ms (Malay)
29.mt (Maltese)
30.yua (Yucatec Maya )
31.no (Norwegian Bokmål)
32.otq (Querétaro Otomi)
33.fa_IR (Persian)
34.pl_PL (Polish )
35.pt_Pt (Portugese )
36.ro_Ro (Romanian )
37.ru_RU (Russian)
38.sr-Cyrl_SP (Serbian (Cyrillic) )
39.sr-Latn_SP Serbian (Latin)
40.sk_SK (Slovak )
41.sl_SL (Slovenian)
42.sv_SE (Swedish)
43.th_TH (Thai )
44.tr_TR (Turkish )

Feel Free to edit and change them.

### CODE REVIEW

**Comment by WardBenjamin** [comment]
Fyi, the file names on these aren't formatted correctly (for example, en_CA). Also, some of these are duplicates of already-existing localizations. (FR, for example).

**Comment by 65001** [comment]
Do you have a list of where I can correctly reformat the file paths ? Should I use the ISO language codes ? 

I didn't see the French localization , sorry.

File: Assets/StreamingAssets/Localization/he_IL.lang [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+save=שמור
+temp_pathfinding_test=מבחן pathfinding
+selection_panel_title=הבחירה הנוכחית
+save_file=שמירת קובץ...
+load_file=לטעון קובץ...
+okay=בסדר
+cancel=ביטול
+filename=שם קובץ
+selection_hp={0} / {1}
+build=לבנות {0}
```
Comment (bjubes) [inline Assets/StreamingAssets/Localization/he_IL.lang]: Build seems to have a brackets mishap

**Comment by empireages** [comment]
The french localization is mine :)
PR #262 .
Also I edited the wiki for translator.
Feel free to check out and edit.
[https://github.com/TeamPorcupine/ProjectPorcupine/wiki/Localisation](Localization Wiki)

File: Assets/StreamingAssets/Localization/he_IL.lang (hunk 1 above)
Comment (65001) [inline Assets/StreamingAssets/Localization/he_IL.lang]: I think I fixed it

**Comment by SanteriHetekivi** [comment]
Finnish localization is also in PR (#183) (mine) (Made 22 hour ago) (Updated 2 hour ago).
@empireages Thanks for info about wiki, just updated my Finnish translation there. :)

**Comment by 65001** [comment]
@empireages  I added all the ones in this page to the wikipage . Should I delete the ones already translated by real people ?

**Comment by SanteriHetekivi** [comment]
@65001 At least script version of Finnish is incomplete (like missing translation for new_world) and my (#183) translation is currently up to date and made by native speaker.

But for languages that have no translation already, these are good starting points.
Or at least they seem that way. :D

**Comment by vogonistic** [comment]
@65001 The translations by real people should always take precedence over machine translations.

**Comment by WardBenjamin** [comment]
Agreed, the ones validated by native speakers should be preferred, since translation tools aren't usually that fantastic.

**Comment by 65001** [comment]
@vogonistic @WardBenjamin  I agree , I'm new to github so I should delete my versions of the files to prevent conflict with the native translations or something along those lines ?

**Comment by vogonistic** [comment]
@65001 That's definitely the easiest option. You can always use it to fill in missing keys in existing translations later.

**Comment by empireages** [comment]
So as a little rule : 
 -if a human translate a thing, it will be better than robot. (90% :) )
 -i will later reorganize the wiki page so we have a good visibility of what is translated, for which version and by robot or not.

I think we can keep the robot translation along homemade one.
(Also I think you can share your tool for future translation key on the wiki for other if you want)

**Comment by 65001** [comment]
@empireages  It's a really simple tool for right now , it takes the current en_US.lang file and translates it ,I need to work on it some more before I upload it to GD or Github and upload it

**Comment by empireages** [comment]
No problem :)
@65001 As you are good at string manipulation, can you checkout the wiki page i've updated and convert your edits if you can, if so it's one thinks less to update :) ?
@quill18  Does an Utility Folder will be added outside of the unity project ?

**Comment by 65001** [comment]
@empireages the update to my edits looks good . I wouldn't say I'm the greatest , I'm not using a well known language ("Small Basic") to do all of this translating . I'll upload a working version as soon as I can

**Comment by JosephBywater** [comment]
@65001 I think I used Small Basic when it first came out! :D

**Comment by 65001** [comment]
@JosephBywater  That's cool :) . I'm using the LD extension that's written in C# to translate everything across ... I'm just waiting for quill's video on C# to come out so I can start learning it

**Comment by quill18** [comment]
You people are maniacs.  Merged!

**Pull request merged**